react/useProductImpression.tsx

```diff
@@ -22,25 +27,29 @@ const sendImpressionEvents = (
   }))
   push({
     event: 'productImpression',
-    list: 'Shelf',
+    list: listName || 'Shelf',
```

Comment:
will there ever be a case `listName` is falsy? If so, is 'Shelf' a good default value?